pkg/sql/executor_statement_metrics.go

```diff
@@ -189,7 +186,11 @@ func (ex *connExecutor) recordStatementSummary(
 	}
 
 	nodes := util.CombineUnique(getNodesFromPlanner(planner), []int64{nodeID})
```

Comment:
Additionally: ExtractNodesFromSpan returns an unordered set already (intsets.Fast). Then ForEach converts it into a sequence. Then the caller uses util.CombineUnique to transform the sequence back into a set. This is just silly and these steps should be combined.

(cc @j82w )